script/barrelize.js

```diff
@@ -42,6 +42,9 @@ const ignoredTypes = [
   'BackgroundReplacementVideoFrameProcessorDelegate',
   'BackgroundReplacementFilter',
 
+  // These are classes we don't want to export
+  'DefaultStatsCollector',
```

Comment:
Shouldn't this be StatsCollector since you rename it?

Reply:
Right, I forgot change this in the second commit. Updated.